scoreboard/models.py

```diff
@@ -46,17 +46,18 @@ class Meta:
         abstract = True
 
     # these need to be nullable otherwise initial player/clan creation fails
-    lowest_turncount       = models.IntegerField(null=True)
-    fastest_realtime       = models.DurationField(null=True)
-    max_conducts_in_1_game = models.IntegerField(null=True)
-    max_ach_in_1_game      = models.IntegerField(null=True)
-    min_score              = models.IntegerField(null=True)
-    max_score              = models.IntegerField(null=True)
-    longest_streak         = models.IntegerField(null=True)
-    earliest_asc_time      = models.DateTimeField(null=True)
-    unique_deaths          = models.IntegerField(null=True)
-    unique_ascs            = models.IntegerField(null=True)
-    games_over_1000_turns  = models.IntegerField(null=True)
+    # ascension_count        = models.IntegerField(default=0)
+    # games_t1000_count      = models.IntegerField(default=0)
```

Comment:
is the idea that these would now be done with .count() instead of using the leaderboard base fields as a counter?

Reply:
yeah pretty much